### PR TITLE
add fields(code / isActive / mobile) to user model

### PR DESCRIPTION
**Pull request by MohammadKoolivand**
i add some fields to user model for signup route

### CODE REVIEW

File: src/models/User.js [hunk 1]
```diff
@@ -5,10 +5,13 @@ const Schema = mongoose.Schema;
 
 const UserSchema = new Schema(
     {
-        username:  String ,
-        password: String,
+        username: { type: String, required: true },
```
Comment (farshid0594) [inline src/models/User.js]: add unique:true to unique fields

File: src/models/User.js [hunk 2]
```diff
@@ -5,10 +5,13 @@ const Schema = mongoose.Schema;
...
+        password: { type: String, required: true },
+        mobile: { type: String, required: true },
         name: String,
```
Comment (farshid0594) [inline src/models/User.js]: set max length to name field

File: src/models/User.js [hunk 3]
```diff
@@ -5,10 +5,13 @@ const Schema = mongoose.Schema;
...
         tokens: [{ text: String, expired: String }],
+        code: { codeNo: Number, expired: String },
```
Comment (farshid0594) [inline src/models/User.js]: change codeNo to text .

File: src/models/User.js [hunk 4]
```diff
@@ -5,10 +5,13 @@ const Schema = mongoose.Schema;
 
 const UserSchema = new Schema(
     {
-        username:  String ,
-        password: String,
+        username: { type: String, required: true },
+        password: { type: String, required: true },
+        mobile: { type: String, required: true },
```
Comment (farshid0594) [inline src/models/User.js]: add index:true to fields need to index.

**Review by farshid0594 — CHANGES_REQUESTED** [review]
review comments

File: src/models/User.js (hunk 1 above)
Comment (MohammadKoolivand) [reply src/models/User.js]: added to username and mobile

File: src/models/User.js (hunk 4 above)
Comment (MohammadKoolivand) [reply src/models/User.js]: added index to username

**Review by farshid0594 — APPROVED** [review]

**Review by amirhnr — APPROVED** [review]

**Pull request merged**